### ISSUE TITLE
Wrong predictions after converting from tf2_image_retraining.ipynb to tfjs_graph_model

### ISSUE THREAD

**Issue by vjuss** [tensorflow/tfjs#3717]
TensorFlow.js version: 2.0

Browser version: Chrome 

Problem: Wrong results after conversion

Code to reproduce the bug / link to feature request:

https://github.com/tensorflow/hub/tree/master/tensorflow_hub/tools/make_image_classifier
https://colab.research.google.com/github/tensorflow/hub/blob/master/examples/colab/tf2_image_retraining.ipynb

I have used make_image_classifier to make my own classifier and converted that to Tensorflow.js with success, giving me confidence rates like A: 0.67XXX, B: 0.10XXX, C: 0.23XXXX (together 1.0)

I want to do the same with the corresponding Colab notebook to have more control but if I use it with default settings, I get weird predictions after conversion then, like:

A: 0.09XXX, B: 2.69XXX, C: 0.77XXXX

I wonder why this is happening - how can I use the notebook and convert succesfully?

For both I used this conversion format from https://github.com/tensorflow/tfjs/tree/master/tfjs-converter:

tensorflowjs_converter
--input_format=tf_saved_model
--output_format=tfjs_graph_model
--signature_name=serving_default
--saved_model_tags=serve
/mobilenet/saved_model
/mobilenet/web_model

**Comment by pyu10055** [comment]
@vjuss can you share the saved models from both make_image_classifier script and colab?

**Comment by vjuss** [comment]
@pyu10055 I'm sorry to reply this late! What eventually happened was that I followed advice from this thread https://stackoverflow.com/questions/55849309/retrain-image-detection-with-mobilenet/56340676#56340676 (by you, for example!) and used retrain.py script for my training, as I realized that make_image_classifier was more optimized for TF Lite. I could try to provide the make_image_classifier and colab models for further research - now in the middle of a final project deadline so will look into that once this is over!
